microdata.py

```diff
@@ -115,6 +127,15 @@ def __eq__(self, other):
     def __repr__(self):
         return self.string
 
+    @staticmethod
+    def get_domain(url_string):
```

Comment:
Perhaps this should be replaced with something using `urlparse`, which returns a handy named tuple:
```python
>>> urlparse('https://example.com/foo/bar')
ParseResult(scheme='https', netloc='example.com', path='/foo/bar', params='', query='', fragment='')
```

Reply:
I used `urlparse` inside this method instead of manually splitting and joining but `urlparse` can't be a drop-in replacement for what I want to achieve here:

```
>>> urlparse("github.com/edsu/microdata/pull/32/files")
ParseResult(scheme='', netloc='', path='github.com/edsu/microdata/pull/32/files', params='', query='', fragment='')
```

Reply:
urlparse expects a URL:

```
>>> urlparse("https://github.com/edsu/microdata/pull/32/files")
ParseResult(scheme='https', netloc='github.com', path='/edsu/microdata/pull/32/files', params='', query='', fragment='')
```

Reply:
I'm not really knowledgeable about the field here but my point is, are we sure to never encounter malformed-URLs ? I get it that urllib expects well-formed URLs but shouldn't a microdata parser be more forgiving? Hence I manually check and adapt in case the string leaves out the protocol part.